parlai/crowdsourcing/utils/analysis.py

```diff
@@ -105,7 +147,8 @@ def setup_args(cls):
         )
         return parser
 
-    def __init__(self, opt):
+    def __init__(self, opt: Dict[str, Any]):
+        super().__init__(opt)
```

Comment:
@mojtaba-komeili I've added a `super().__init__(opt)` call here, since I imagine that your subclasses might want to use the `output_folder` attribute, but lemme know if there's a reason why this might not be desired